sky/skylet/providers/lambda_cloud/lambda_utils.py

```diff
@@ -86,8 +87,10 @@ def raise_lambda_error(response: requests.Response) -> None:
         resp_json = response.json()
         code = resp_json.get('error', {}).get('code')
         message = resp_json.get('error', {}).get('message')
-    except (KeyError, json.decoder.JSONDecodeError):
```

Comment:
Why are we removing the `KeyError` here?

Reply:
dict.get() doesn't throw KeyError?